hypervisor/include/common/ptdev.h

```diff
@@ -36,12 +36,37 @@ union source_id {
 	} intx_id;
 };
 
+union msi_addr_reg {
+	uint64_t full;
+	struct {
+		uint32_t rsvd_1:2;
+		uint32_t dest_mode:1;
+		uint32_t rh:1;
+		uint32_t rsvd_2:8;
+		uint32_t dest_field:8;
+		uint32_t constant:12;
```

Comment:
what does "constant" mean? Not very clear to me.

Reply:
Those bits are 0xFEE in the MSI address register. So I named it constant. May be rename it something like "fixed" or "addr_base"?

Reply:
base_addr, or hi_addr?